tests/null_object_tests.py

```diff
@@ -87,3 +95,99 @@ class SimplePage(Page):
         self.driver.refresh()
         with hiro.Timeline().scale(10):
             self.assertEqual(page.id_el_changed, NonexistentElement())
+
+    # test object equivalence/nonequivalence test that expected parameters are displayed
+
+    def test_non_existence_eq_element(self):
```

Comment:
The test for object equivalence should test that our overloaded __eq__method evaluates the way we expect it to for objects of the same class.  E.g:
x = NonexistentElement()
y = NonexistentElement('value1', 'value2', 'value3')
assert(x.property1 != y.property1) #Objects have different property values
assert x == y  #Our overloaded method should still return true
z = 1
assert x !=z  #An Int type, or any other Object that's not a NonexistentElement type, should return False